lib/sentry/rails.rb

```diff
@@ -5,6 +5,7 @@
 require "sentry/rails/configuration"
 require "sentry/rails/engine"
 require "sentry/rails/railtie"
+require "sentry/rails/version"
```

Comment:
After renaming the gem to `sentry-rails4` there was an error in `sentry/rails.rb:12`:

```
sentry/rails.rb:12:in `<module:Rails>': uninitialized constant Sentry::Rails::VERSION (NameError)
```